Clarify usage of _sd_alg to fix #266 (#287)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

### Hash Function Claim {#hash_function_claim}

The claim `_sd_alg` indicates the hash algorithm used by the Issuer to generate

> the digests. This claim MAY appear at the top level of the SD-JWT payload, but
> MUST NOT be used in any object nested within the payload. If the  `_sd_alg`
> claim is not present at the top level, a default value of `sha-256` is used.

Reviewer: ```suggestion
the digests. When used, this claim MUST appear at the top level of the SD-JWT payload. It
MUST NOT be used in any object nested within the payload. If the  `_sd_alg`
claim is not present at the top level, a default value of `sha-256` MUST be used.
```
